### PR TITLE
Makes flashes more useful as a utility item rather than a weapon, as they were meant to be

### PR DESCRIPTION
**Pull request by deathride58**
This PR removes the knockdown effect from flashes, but makes up for it by increasing the utility it has by buffing the confusion effect on direct flashes and adding eyeblur, bringing back their status as a utility item.

:cl: deathride58
balance: Flashes no longer knockdown. Instead, they deal eyeblur and have increased confusion
/:cl:

### CODE REVIEW

**Comment by Ragolution** [comment]
I feel like this relegates them to Pepperspray levels of usefulness (see: useless) and that they'll rapidly fall out of favor, considering they'll be generally inferior to every other disable item. On the other hand, this is a positive thing for Revs rounds as it'll make it harder to detect via security cameras (which I like).

Unsure. Could be good.

**Comment by Trilbyspaceclone** [comment]
I don't understand the need for this change in a already border line bad sec item. Pepper spray can be the most robust lethal tool any sec officer can have, cuffs are a need, stun batons for de-rev-ing, and a flash to disarm people quickly and be a hold out hard stun disarm if only for the .1 seconds.
I feel there balanced in the face of Civ use as a disarm without having a ready stun baton.
Making it not do what its intended to do is lame as well seeings how there ment for *attacking* more then defending. Defending is the lame attack witch is a AOE screen blind useful for runing away and making them a bit more likely to not hit you.
TL:DR
Flashes are already the weakest tool sec has
No real reason to change it in a nerf way
This is hugbox-y at best

**Comment by JogBoba** [comment]
I'm in favor of this. The issue is that flashes are easily available to pretty much everyone, not just security. Security has other tools that should really be better than flashbulbs are at downing people. I've recently experienced someone actually good at using flashes and they're honestly more effective than even stun batons in melee if you aren't wearing something that blocks them, and most people don't unless they're a bit powergaming.

**Comment by deathride58** [comment]
@Trilbyspaceclone the primary issue with flashes right now is that they can actually stamcrit in less than a second on top of dealing eye damage along with the ability to knockdown in an AOE radius. They are literally the strongest tool sec has against targets that don't have eye protection at the moment. This PR trades their ability to stamcrit with the ability to partially immobilize and visually impair a target, giving flashes a definitive use against groups and people who have sources of stam regen.

**Comment by Poojawa** [comment]
> Being blinded makes you so tired you can't stand up again.

Like Appendixes, Spess Man all suffer from terrible epliesie

**Comment by Trilbyspaceclone** [comment]
@deathride58 

> the primary issue with flashes right now is that they can actually stamcrit in less than a second on top of dealing eye damage along with the ability to knockdown in an AOE radius.

This is a misleading claim, you make it out to seem they do much in terms of stam harm, as well as its AOE being of witch a KD stun, witch if you played with them you would know is wrong. there AOE just blines and dosn't even harm stam. It's just as good as shining a flash light into there eyes

>They are literally the strongest tool sec has against targets that don't have eye protection at the moment.

There stun baton is much better at stunning and KDing them then a single flash. They can burn out in only 3 uses and commonly do. There ruined with easy by EMP or bombs making them pointless to horde against anyone of threat.
There strongest tool without a dout has to there pepper spray when used in a new mix. 

>This PR trades their ability to stamcrit with the ability to partially immobilize and visually impair a target, giving flashes a definitive use against groups and people who have sources of stam regen.

I see not what you mean, there AOE dosnt affect stam and making it would be hugbox. Again misleading claim of stam crit. Tho I see the point you make its rather loss in facts of how mirror it is and how many items block flashes, how many counters there are. They have to be rather by far the worst sec item and nerfing them only harms Borgs, sec and revs without really any trade off of value.

**Comment by Ragolution** [comment]
After a recent experiment/event on Main I 1000% support this PR.

**Review by silicons — APPROVED** [review]
